src/ValueFormatters/QuantityFormatter.php

```diff
@@ -80,12 +91,22 @@ public function __construct(
 
 		$this->decimalFormatter = $decimalFormatter ?: new DecimalFormatter( $this->options );
 		$this->vocabularyUriFormatter = $vocabularyUriFormatter;
+		$this->quantityWithUnitFormat = $quantityWithUnitFormat ?: '$1 $2';
 
 		// plain composition should be sufficient
 		$this->decimalMath = new DecimalMath();
 	}
 
 	/**
+	 * @since 0.6
+	 *
+	 * @return string
+	 */
+	protected function getQuantityWithUnitFormat() {
```

Comment:
Why do we need this method? Why is it protected? What can be achieved by overriding it?